parallel-state.js

```diff
@@ -0,0 +1,43 @@
+var Nanobus = require('nanobus')
+var assert = require('assert')
+
+module.exports = Parallelstate
+
+function Parallelstate (transitions) {
+  if (!(this instanceof Parallelstate)) return new Parallelstate(transitions)
```

Comment:
Because we're always calling `new` in our `index.js` constructor, we don't need this line :grin:

Reply:
Hey! I just left in case someone else to try to call without `new`. But I'll ✂️ it out!